cmd/logger.go

```diff
@@ -17,78 +17,65 @@
 package cmd
 
 import (
+	"encoding/json"
 	"fmt"
-	"io/ioutil"
+	"os"
 	"path"
+	"path/filepath"
 	"runtime"
 	"strings"
+	"time"
 
-	"github.com/Sirupsen/logrus"
 	"github.com/minio/mc/pkg/console"
 	"github.com/minio/minio/pkg/errors"
 )
 
 var log = NewLogger()
 
-// LogTarget - interface for log target.
-type LogTarget interface {
-	Fire(entry *logrus.Entry) error
-	String() string
-}
+type levelType int8
 
-// BaseLogTarget - base log target.
-type BaseLogTarget struct {
-	Enable    bool `json:"enable"`
-	formatter logrus.Formatter
-}
+// Enumerated level types
+const (
+	ERROR levelType = iota // 0
+	FATAL                  // 1
+)
 
-// Logger - higher level logger.
-type Logger struct {
-	logger        *logrus.Logger
-	consoleTarget ConsoleLogger
-	targets       []LogTarget
-	quiet         bool
+func (l levelType) String() string {
+	levels := [...]string{
+		"ERROR",
+		"FATAL",
+	}
+	return levels[l]
```

Comment:
Its better to check and return than fixed indexing (to avoid errors in future i.e. order of enums may change)

```go
switch l {
case ERROR:
    return "ERROR"
...
}
```

Reply:
Done.